File: MODUL 3/PRAK304-2310817220011-ReginaSilvaMaharatini.c
#include <stdio.h>

int main(){
    int num;
    printf("Input\n");
    scanf("%d", &num);
    printf("\nOutput");

    if(num == 0){
        printf("Nol");
    } else if(num >= 1 && num <= 9){
        printf("Satuan");
    } else if(num >= 10 && num <= 19){
        printf("Belasan");
    } else if(num >= 20 && num <= 99){
        printf("Puluhan");
    } else if(num >= 100){
        printf("Anda Menginput Melebihi Limit Bilangan");
    }
    return 0;
}

File: MODUL 3/PRAK304-2310817220011-ReginaSilvaMaharatini.py
num = int(input("Input\n"))
print("\nOutput")

if num == 0:
    print("Nol")
elif 1 <= num <= 9:
    print("Satuan")
elif 10 <= num <= 19:
    print("Belasan")
elif 20 <= num <= 99:
    print("Puluhan")
elif num >= 100:
    print("Anda Menginput Melebihi Limit Bilangan")
